utils/lib/Quickstart.ts

```diff
@@ -198,6 +198,20 @@ class Quickstart {
     };
   }
 
+  public transformForArtifact(): QuickstartConfig | { authors: { name: string }[] } {
```

Comment:
TODO:
- `quickstart_uuid` over `id`
- `icon_url` over `icon`
- `authors` is a list of names